espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/UiautomatorPageSource.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.handlers
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import io.appium.espressoserver.lib.helpers.InteractionHelper
+import io.appium.espressoserver.lib.model.AppiumParams
+import java.io.File
+
+class UiautomatorPageSource : RequestHandler<AppiumParams, String> {
+
+    @Throws(AppiumException::class)
+    override fun handleInternal(params: AppiumParams): String {
+        var dumpXml = ""
+        var dumpView: File? = null
+
+        try {
+            val uiDevice = InteractionHelper.getUiDevice()
```

Comment:
this intermediate variable is not needed

Reply:
fixed